Placeeo/Controller/LocationManager.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  LocationManager.swift
+//  Placeeo
+//
+//  Created by Pasha Berger on 03.10.2021.
+//
+
+import Foundation
+import CoreLocation
+
+class LocationManager: NSObject, CLLocationManagerDelegate {
+    static let shared = LocationManager()
+    
+    let manager = CLLocationManager()
+    
+    var completion: ((CLLocation) -> Void)?
+    
+    public func getUserLocation(completion: @escaping ((CLLocation) -> Void)) {
+        self.completion = completion
+        manager.requestWhenInUseAuthorization()
+        manager.delegate = self
+        manager.startUpdatingLocation()
+        
+    }
+    public func resolveLocationName(with location: CLLocation, completion: @escaping ((String?)->Void)) {
+        let geocoder = CLGeocoder()
+        geocoder.reverseGeocodeLocation(location, preferredLocale: .current) {placemarks,
+                                                                              error in
```

Comment:
please follow the common code style described in https://github.com/raywenderlich/swift-style-guide